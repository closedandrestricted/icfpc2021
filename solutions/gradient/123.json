{"vertices":[[18,48],[20,110],[27,73],[35,63],[29,19],[38,88],[39,32],[38,69],[50,51],[46,40],[53,67],[52,77],[58,87],[50,106],[59,63],[59,77],[62,81],[57,40],[61,116],[69,61],[75,76],[72,80],[68,40],[76,61],[79,70],[76,86],[73,107],[62,49],[78,38],[90,66],[77,22],[96,87],[100,60],[93,53],[106,113],[108,36]]}