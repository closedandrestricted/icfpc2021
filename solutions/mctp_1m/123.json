{"vertices":[[14,85],[9,43],[28,72],[28,70],[57,50],[30,62],[48,37],[35,64],[63,50],[54,44],[47,55],[49,65],[48,75],[50,96],[52,60],[51,72],[48,68],[51,55],[37,95],[51,70],[52,79],[47,78],[54,45],[53,77],[47,84],[49,85],[46,105],[54,56],[55,35],[38,75],[40,33],[46,102],[29,68],[27,87],[26,83],[23,109]]}