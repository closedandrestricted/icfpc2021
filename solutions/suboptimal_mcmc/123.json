{"vertices":[[19,106],[17,48],[29,67],[36,91],[32,24],[41,64],[41,38],[40,70],[53,56],[49,45],[55,69],[54,79],[62,88],[49,105],[62,67],[61,77],[64,81],[59,40],[59,116],[72,64],[76,75],[73,79],[69,43],[78,60],[80,69],[72,86],[71,107],[76,51],[78,38],[93,62],[77,22],[98,84],[103,57],[94,52],[108,111],[109,35]]}