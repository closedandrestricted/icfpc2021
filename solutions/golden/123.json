{"vertices":[[20,108],[16,48],[28,72],[35,91],[31,24],[40,64],[40,38],[39,70],[52,56],[48,45],[54,69],[53,79],[61,88],[48,105],[61,67],[60,77],[63,81],[58,40],[58,116],[71,64],[75,75],[72,79],[69,43],[77,60],[79,69],[70,86],[70,107],[76,51],[78,38],[91,62],[77,22],[97,84],[102,58],[93,52],[107,111],[108,35]]}